models/cable/R/write.config.CABLE.R

```diff
@@ -120,6 +120,6 @@ write.config.CABLE <- function(defaults, trait.values, settings, run.id) {
   config.text <- gsub("@OUTFILE@", paste0("out", run.id), config.text)
   
   #-----------------------------------------------------------------------
-  config.file.name <- paste0("CONFIG.", run.id, ".txt")
+  config.file.name <- cable.nml
```

Comment:
cable.nml will need to be in quotes since it's a string not the name of a variable